docs/embedded/tutorials/doc-processing-acs.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Document Processing with ACS
+description: Enabling document processing with ACS
+ms.date: 02/26/2024
+ms.localizationpriority: high
+---
+
+# Enabling Document Processing with ACS
+
+## Utilizing Azure Cognitive Services
```

Comment:
Proper markdown formatting is to include a line break between each heading & paragraph. See existing docs for reference.